### ISSUE TITLE
Ductal bile plugs

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7562]
**Preferred term label:**
Ductal bile plugs

**Synonyms**

**Definition (free text, please give PubMed ID)**
Bile thrombi that form an obstruction (plug) in a dilated bile duct
PMID: 2028872  PMID: 22152905

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617394 
PMID: 27469900  4/7

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
